calcite-qml/demo/arcgisruntime.pri

```diff
@@ -25,6 +25,6 @@ iniLine = $$cat($${iniPath}, "lines")
 dirPath = $$find(iniLine, "InstallDir")
 cleanDirPath = $$replace(dirPath, "InstallDir=", "")
 priLocation = $$replace(cleanDirPath, '"', "")
-!include($$priLocation/sdk/ideintegration/arcgis_runtime_qml.pri) {
```

Comment:
@jared-2016 I get your last changes from v.next, I don't know why these changes are detected in GitHub.

Reply:
Why `calcite-qml` and not just `calcite`?

Reply:
Oh the folder containing calcite demo? it just slipped my radar. You can put that change in this PR. Sorry for that.

Reply:
> @jared-2016 I get your last changes from v.next, I don't know why these changes are detected in GitHub.

Yeah that's interesting. This was what I changed. 🤷

Reply:
Done